nautobot/docs/release-notes/version-1.0.md

```diff
@@ -16,36 +16,61 @@ Users migrating from NetBox to Nautobot should also refer to the ["Migrating fro
 - [#122](https://github.com/nautobot/nautobot/pull/122) - Improved installation flow for creating nautobot user and virtualenv
 - [#131](https://github.com/nautobot/nautobot/pull/131) - Replaced PostgreSQL-specific ArrayField with a more generic JSONArrayField
 - [#142](https://github.com/nautobot/nautobot/pull/142) - Converted various config validation checks into proper Django checks
+- [#149](https://github.com/nautobot/nautobot/issues/149) - Unify optional settings documentation for `REMOTE_AUTH*/SOCIAL_AUTH*`
+- [#159](https://github.com/nautobot/nautobot/issues/159) - Update documentation for external authentication SSO Backend to get a proper install
 - [#180](https://github.com/nautobot/nautobot/pull/180) - Revised available Invoke tasks for simplicity and maintainability
+- [#208](https://github.com/nautobot/nautobot/pull/208) - Custom fields model refactor
+- [#216](https://github.com/nautobot/nautobot/pull/216) - Update install docs to address inconsitent experience w/ `$PATH`
+- [#235](https://github.com/nautobot/nautobot/pull/235) - Update restart docs to include worker
+- [#236](https://github.com/nautobot/nautobot/pull/236) - Add `post_upgrade` to developer docs and add `invoke post-upgrade`
 
 ### Removed
 
 - [#124](https://github.com/nautobot/nautobot/pull/124) - Removed incorrect statement from feature request template
 - [#161](https://github.com/nautobot/nautobot/pull/161) - Removed leftover references in documentation to `RQ_DEFAULT_TIMEOUT`
+- [#188](https://github.com/nautobot/nautobot/pull/189) - Remove `CSRF_TRUSTED_ORIGINS` from core settings
+- [#189](https://github.com/nautobot/nautobot/pull/189) - Remove all references to `settings.BASE_PATH`
 
 ### Fixed
 
+- [#26](https://github.com/nautobot/nautobot/issues/26) - `nautobot-server runserver` does not work using `poetry run`
+- [#58](https://github.com/nautobot/nautobot/issues/58) - GraphQL Device Query - Role filter issue
 - [#76](https://github.com/nautobot/nautobot/issues/76) - Cable paths could not be traced through circuits
 - [#95](https://github.com/nautobot/nautobot/issues/95) - Plugin load errors under Gunicorn
-- [#128](https://github.com/nautobot/nautobot/pull/128) - Overview of usage for the `nautobot-netbox-importer` plugin could be mistaken for full instructions
+- [#127](https://github.com/nautobot/nautobot/issues/127) - SSL error: decryption failed or bad record mac & SSL SYSCALL error: EOF detected
 - [#132](https://github.com/nautobot/nautobot/issues/132) - Generated `nautobot_config.py` did not include a trailing newline
 - [#134](https://github.com/nautobot/nautobot/issues/134) - Missing venv activation step in install guide
+- [#135](https://github.com/nautobot/nautobot/issues/135) - Custom field Selection value name change causes data inconsistency
+- [#137](https://github.com/nautobot/nautobot/issues/137) - Support Custom Field Name Changes
```

Comment:
This was the name of the issue, but the change is to _explicity disallow_ changing the name of a custom field.

Reply:
LOL yeah I just followed suit by copying the issue title.